docs/book/src/introduction.md

```diff
@@ -20,8 +20,9 @@ previous cluster managers such as [kops](https://github.com/kubernetes/kops) and
 
 <h1>Cluster API Provider IBM Cloud documentation versions</h1>
 
-This book documents Cluster API Provider IBM Cloud v0.8. For other versions please see the corresponding documentation:
+This book documents Cluster API Provider IBM Cloud v0.9. For other versions please see the corresponding documentation:
 * [main.cluster-api-ibmcloud.sigs.k8s.io](https://main.cluster-api-ibmcloud.sigs.k8s.io)
+* [release-0-9.cluster-api-ibmcloud.sigs.k8s.io](https://release-0-9.cluster-api-ibmcloud.sigs.k8s.io/)
```

Comment:
this link doesn't work, not sure netlify really got fixed or not!

Reply:
nope not yet fixed and same issue with 0.8
I am reaching out to them via slack again to get this fixed asap.

Reply:
It is not resolved yet, the latest I could find out is it is failing post this change - https://github.com/kubernetes/k8s.io/pull/5745

previous issue - https://github.com/kubernetes/org/issues/4318
open issue - https://github.com/kubernetes/org/issues/5136

Reply:
release-0.8 url got fixed, I am not able to see release-0.9 yet in the list of branch subdomains yet.

Reply:
release-0.9 fixed.